.github/workflows/ci.yml

```
@@ -282,13 +282,74 @@ jobs:
+          # "skills::register")`) is a pre-existing reference (the prefix
+          # is still hidden from MCP discovery even though no one fires it
+          # any more) and is intentionally not matched.
+          matches=$(grep -rn 'function_id:.*"skills::register"' \
+            --include='*.rs' . \
+            | grep -v '.worktrees/' \
+            | grep -v 'target/' \
+            | grep -v '.git/' \
+            || true)
+          if [ -n "$matches" ]; then
+            echo "::error::\"skills::register\" TriggerRequest found in Rust sources — \
+the iii-directory migration replaced it with on-disk skill.md (see \
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)